composer.json

```diff
@@ -20,6 +20,8 @@
     },
     "require": {
         "php": "~7.2 || ~7.3 || ~7.4",
+        "helmich/typo3-typoscript-lint": "^2.0",
+        "squizlabs/php_codesniffer": "^3.5",
```

Comment:
For dev dependencies, I prefer to point to the latest bugfix release, i.e., in this case ^2.0.1 and ^3.5.3 so we won't get the added complexity of bugs with older versions.